lib/pivotal-tracker/account_membership.rb.orig

```diff
@@ -0,0 +1,31 @@
+class PivotalTracker::AccountMembership
```

Comment:
`.orig` files are a side affect of `git mergetool` and should not be committed.  This is usually caused by running `git add .` instead of specifically staging the changes you want staged.

Please `git rm` this file, commit, then squash it into the existing commits.